packages/cli/src/index.ts

```diff
@@ -109,8 +110,22 @@ const main = async () => {
     return 1;
   }
 
+  const isNoColor = argv['--no-color'];
+
+  if (
+    process.env.FORCE_COLOR === '0' ||
+    process.env.NO_COLOR === '1' ||
+    isNoColor
+  ) {
+    process.env.NO_COLOR = '1';
+    chalk.level = 0;
+  }
+
   const isDebugging = argv['--debug'];
-  const output = new Output(process.stderr, { debug: isDebugging });
+
+  const output = new Output(process.stderr, {
+    debug: isDebugging,
```

Comment:
I think we should store the `noColor` param here on the Output object, so that we can properly test the scenarios in the unit tests without messing around with the env vars.